server/src-lib/Hasura/Server/API/PGDump.hs

```diff
@@ -87,7 +87,10 @@ execPGDump b ci = do
 
     notifyTriggerRegex =
       let regexStr :: String =
-            "^CREATE TRIGGER \"?notify_hasura_.+\"? AFTER [[:alnum:]]+ "
+        -- "notify_hasura_" is here for backwards compatibility. Prior to "hasura_"
+        -- pg functions created by hasura for event triggers used "notify_hasura"
+        -- These changes are also documented on the method pgIdenTrigger
```

Comment:
@jberryman I'm not sure if this is good enough. It'd great if you could help me improve on this.